### PR TITLE
barman: init 2.11

### PR DESCRIPTION
**Pull request by freezeboy**
###### Motivation for this change

New package

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/barman/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    sha256 = "0w5lh4aavab9ynfy2mq09ga6j4vss4k0vlc3g6f5a9i4175g9pmr";
+  };
+
+  propagatedBuildInputs = [ dateutil argh psycopg2 boto3 argcomplete ];
+
+  # Tests are not present in tarball
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://www.2ndquadrant.com/fr/resources/barman/";
```
Comment (symphorien) [inline pkgs/tools/misc/barman/default.nix:21]:
```suggestion
    homepage = "https://www.2ndquadrant.com/en/resources/barman/";
```

File: pkgs/tools/misc/barman/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  src = fetchurl {
+    url = "mirror://sourceforge/pgbarman/${version}/barman-${version}.tar.gz";
+    sha256 = "0w5lh4aavab9ynfy2mq09ga6j4vss4k0vlc3g6f5a9i4175g9pmr";
+  };
+
+  propagatedBuildInputs = [ dateutil argh psycopg2 boto3 argcomplete ];
+
+  # Tests are not present in tarball
+  doCheck = false;
+
```
Comment (jonringer) [inline pkgs/tools/misc/barman/default.nix:19]: This catches a lot of failures, and checkPhase on buildPythonApplication maps to installCheckPhase on mkDerivation.
```suggestion
  checkPhase = ''
    $out/bin/barman --help > /dev/null
  '';

```

File: pkgs/tools/misc/barman/default.nix (hunk 1 above)
Comment (freezeboy) [reply pkgs/tools/misc/barman/default.nix:21]: Good catch, bored of automatic redirects ....

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 97880 [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>1 package built:</summary>  <ul>    <li>barman</li>  </ul></details>

**Pull request merged**